### ISSUE TITLE
Testing updated plotting functions

### ISSUE THREAD

**Issue by dsy109** [dsy109/mixtools#2]
@drh20drh20 Recently, @Kedai-Cheng has uploaded testable versions of updated graphics functions for `mixtools`.  These are located under the branch he created.  I believe you were a developer, or at least a contributor, for some of the original functions.  When convenient, would you please have a look at the updated functions listed below to see if they are consistent with the desired output of the original functions?  Please inform us of bugs as well as noting any additional functionality you would like to see.  Note that we have not settled on an official naming convention for these updated functions.  Thanks.

[plotly.FDR.R](https://github.com/dsy109/mixtools/blob/Updated-Functions/Updated-Functions/plotly.FDR.R)
[plotly.npEM.R](https://github.com/dsy109/mixtools/blob/Updated-Functions/Updated-Functions/plotly.npEM.R)
[plotly.spEMN01.R](https://github.com/dsy109/mixtools/blob/Updated-Functions/Updated-Functions/plotly.spEMN01.R)
[plotlyexpRMM.R](https://github.com/dsy109/mixtools/blob/Updated-Functions/Updated-Functions/plotlyexpRMM.R)
[plotlyseq.npEM.R](https://github.com/dsy109/mixtools/blob/Updated-Functions/Updated-Functions/plotlyseq.npEM.R)
[plotlyspRMM.R](https://github.com/dsy109/mixtools/blob/Updated-Functions/Updated-Functions/plotlyspRMM.R)
[plotlyweibullRMM.R](https://github.com/dsy109/mixtools/blob/Updated-Functions/Updated-Functions/plotlyweibullRMM.R)

**Comment by drh20drh20** [comment]
@dsy109 You're correct:  I helped develop npEM and spEM functions.  I believe that spRMM was Didier's group though; I didn't work on the right-censored data.  I'll have a look.
